src/stores/RegistriesStore.ts

```diff
@@ -34,7 +34,9 @@ type NetworkTypesMap = {
 const networkTypesMap: NetworkTypesMap = {
 	centrifuge: {
 		alias: 'centrifuge-chain',
-		chains: {}
+		chains: {
+			centrifuge_amber: 'centrifuge-chain-amber'
```

Comment:
It seems this is not record in the [API](https://github.com/polkadot-js/api/tree/master/packages/types-known/src) yet?

Reply:
If I read the code correctly, it uses `centrifuge` as the lookup in the API's known types. `centrifuge-chain-amber` might be used at a later point as we will use it for breaking changes. Do you think we should add separate types for `centrifuge-chain-amber` already to the API?

Reply:
understood, no need to add separate types for now, the types of `centrifuge-chain` will be included with this structure. If you amber chain has breaking changes then you may add them into API to overwrite the main net's types.